### PR TITLE
Issue 110

### PR DESCRIPTION
**Pull request by opsationby**

### CODE REVIEW

File: main/adv_post.c [hunk 1]
```diff
@@ -126,8 +141,20 @@ parse_adv_report_from_uart(const re_ca_uart_payload_t *const msg, adv_report_t *
     return err;
 }
 
-void
-adv_post_send(void *arg)
+static void
```
Comment (ojousima) [inline main/adv_post.c:144]: Please add a comment describing when this function gets called and what *arg is expected to contain.

File: main/adv_post.c [hunk 2]
```diff
@@ -126,8 +141,20 @@ parse_adv_report_from_uart(const re_ca_uart_payload_t *const msg, adv_report_t *
...
+adv_post_send_ack(void *arg)
+{
+    // Do something
+}
+
+static void
+adv_post_send_device_id(void *arg)
```
Comment (ojousima) [inline main/adv_post.c:151]: Please add a comment describing when this function gets called and what *arg is expected to contain

**Review by ojousima — CHANGES_REQUESTED** [review]

**Comment by ojousima** [comment]
Add to whitelist

**Comment by ojousima** [comment]
Add to whitelist

**Comment by ojousima** [comment]
retest this please

**Comment by ojousima** [comment]
retest this please

**Comment by ojousima** [comment]
Tested on Gateway A1 with nRF v0.4.0 by @theBASTI0N.

**Review by ojousima — APPROVED** [review]

**Review by TheSomeMan — APPROVED** [review]

**Pull request merged**